Makefile

```diff
@@ -273,7 +273,10 @@ init-db-test-from-prod: db_prod_dump.trig init-db-test-empty ## init local datab
 	@curl -X POST -H "Content-Type: application/trig" --data-binary "@${CURRENT_DIR}/db_prod_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
 
 .PHONY: init-db-test-from-ls-test-server
-init-db-test-from-ls-test-server: db_ls_test_server_dump.trig init-db-test-empty ## init local database with data from ls-test-server
+init-db-test-from-ls-test-server: db_ls_test_server_dump.trig init-db-test-from-ls-test-server-trig-file ## init local database with data from ls-test-server
```

Comment:
Remove this completely, because:
1. run `db_ls_test_server_dump.trig` to get the data from server
2. run `init-db-test-from-ls-test-server-trig-file` to load the data from the file

Reply:
isn't it nicer to have a target to combine those steps for you?

Reply:
It is very elegant indeed. Skip my above comment. Looks good.

Reply:
"very elegant indeed" almost sounds like irony :P